pkg/apis/monitoring/v1/podmonitor_types.go

```diff
@@ -31,7 +31,20 @@ const (
 // +k8s:openapi-gen=true
 // +kubebuilder:resource:categories="prometheus-operator",shortName="pmon"
 
-// PodMonitor defines monitoring for a set of pods.
+// The `PodMonitor` custom resource definition (CRD) allows to declaratively define how a dynamic set of pods should be monitored.
+// Which pods are selected to be monitored with the desired configuration is defined using label selections.
+// This allows an organization to introduce conventions around how metrics are exposed, and then following these conventions new pods are automatically discovered, without the need to reconfigure the system.
+//
+// A `Pod` is a collection of one or more containers which can expose Prometheus metrics on a number of ports.
```

Comment:
```suggestion
```

I think we don't need to mention what a pod is :)

Reply:
I will fix this in the next PR.